### ISSUE TITLE
Can't import from FreeOTP to FreeOTPPlus

### ISSUE THREAD

**Issue by BenjieFiftysix** [helloworld1/FreeOTPPlus#242]
Selecting "backup" from FreeOTP (freeotp-android) 2.0 (24) results in a file named `externalBackup`.
I tried importing this file into FreeOTP+ (FreeOTPPlus) 3.0 (21) using both "import JSON format" and "key URI format". Both result in the error message "Failed to import [format]. The file is malformed".

**Comment by Maschmalow** [comment]
Hey, I've made a tool to convert FreeOTP backups to a URI list compatible with FreeOTP+. you can find it here: https://github.com/Maschmalow/freeotp-android

**Comment by Benjamin-Loison** [comment]
@Maschmalow Would you mind [opening `Issues` on your repository](https://github.com/Maschmalow/freeotp-android/settings#features)? I would like to discuss about proposing a trustful APK (built on F-Droid transparent infrastructure for instance, as [I am unable to build the app for my Fairphone 4 running LineageOS](https://github.com/Benjamin-Loison/convert-apk/issues/1)) and wanted to share with people [this script](https://gitea.lemnoslife.com/Benjamin_Loison/Verified_repositories/issues/5) easing reviewing your changes.

**Comment by johnnynoone** [comment]
[Aegis](https://github.com/beemdevelopment/Aegis) now supports importing from FreeOTP 2. Perhaps their code could be reused?

**Comment by helloworld1** [comment]
Yes indeed. Thanks!. Their "app/src/main/java/com/beemdevelopment/aegis/ui/tasks/PBKDFTask.java" "app/src/main/java/com/beemdevelopment/aegis/importers/FreeOtpPlusImporter.java" are likely useful to port over here.
